test/rqg/mzcompose.py

```diff
@@ -27,29 +27,40 @@ class Dataset(Enum):
     SIMPLE = 1
     DBT3 = 2
 
+    def files(self) -> list[str]:
+        match self:
+            case Dataset.SIMPLE:
+                return ["simple.sql"]
+            case Dataset.DBT3:
+                return ["dbt3-ddl.sql", "dbt3-s0.0001.dump"]
```

Comment:
These files are not part of this PR. Where are they located? Where they already introduced with another PR or ar they outside of the repo?

Reply:
For license purity reasons, they are in the MaterializeInc/RQG repository.